src/UI/HTMLTooltip.js

```diff
@@ -54,18 +56,16 @@ export class HTMLTooltip {
     /** @type {HTMLTooltipOptions} */
     options;
     /**
-     * @param config
      * @param inputType
      * @param getPosition
      * @param {HTMLTooltipOptions} options
      */
-    constructor(config, inputType, getPosition, options) {
+    constructor(inputType, getPosition, options) {
```

Comment:
I noticed `config` was never used, so I removed it here (made it easier to instantiate in the debug part)